IronFoundry.Container.Test/Utilities/FileSystemEffectiveAccessComputerTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.IO;
+using System.Security.AccessControl;
+using System.Security.Principal;
+using Xunit;
+
+namespace IronFoundry.Container.Utilities
+{
+    public class FileSystemEffectiveAccessComputerTests : IDisposable
+    {
+        TempFile TestFile { get; set; }
+        FileSystemEffectiveAccessComputer EffectiveAccess { get; set; }
+        IdentityReference CurrentIdentity { get; set; }
+
+        public FileSystemEffectiveAccessComputerTests()
+        {
+            TestFile = new TempFile(Path.GetTempPath());
+            EffectiveAccess = new FileSystemEffectiveAccessComputer();
+            CurrentIdentity = WindowsIdentity.GetCurrent().User;
+        }
+
+        public void Dispose()
+        {
+            TestFile.Dispose();
+        }
+
+        public class ComputeAccess : FileSystemEffectiveAccessComputerTests
+        {
+            [Fact(Skip = "Fails on domain joined machines")]
```

Comment:
Yeah, I need a domain joined machine to investigate more.  One likely problem is the creation of the user context which states:

The AuthzInitializeContextFromSid function creates a user-mode client context from a user security identifier (SID). **Domain SIDs retrieve token group attributes from the Active Directory.**

There is another way to create the context using the user token which presumably would not need to go to AD to fetch the security groups.  However, I don't have a domain joined machine at the moment to test this.